File: Gym.kt
package com.example.mygymapplication.model

import java.time.LocalDate
import java.time.LocalTime

data class Gym (
    val id:String,
    var className:String,
    var trainerName:String,
    var description:String,
    var startTime:String,
    var endTime:String,
    var availableSpots:Int
)

File: non-native/WithDatabase/gym.dart
class Gym {
  final String id;
  String className;
  String trainerName;
  String description;
  String startTime;
  String endTime;
  int availableSpots;

  Gym({
    required this.id,
    required this.className,
    required this.trainerName,
    required this.description,
    required this.startTime,
    required this.endTime,
    required this.availableSpots,
  });
}
